src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersForm.tsx

```diff
@@ -50,114 +52,229 @@ const TableHeadersForm = ({
+                    <DragDropContext
+                      onDragEnd={result => {
+                        handleDragEnd(form, result);
+                        toggleGroupDraggingActive();
+                      }}
+                      onDragStart={() => {
+                        toggleGroupDraggingActive();
+                      }}
                     >
-                      name="rowGroups"
-                      legend="Row groups"
-                      groupLegend="Row group"
-                    />
-                  </div>
-
-                  <div className="govuk-!-margin-bottom-2">
-                    <FormFieldSortableListGroup<
-                      PickByType<TableHeadersConfig, Filter[][]>
+                      <FormGroup className="govuk-!-margin-bottom-4">
+                        <TableHeadersAxis<
+                          PickByType<TableHeadersConfig, Filter[][]>
+                        >
+                          id="columnGroups"
+                          legend="Move column headers"
+                          name="columnGroups"
+                          onMoveGroupToOtherAxis={groupIndex => {
+                            handleMoveGroupToOtherAxis(
+                              'columnGroups',
+                              groupIndex,
+                              form,
+                            );
+                          }}
+                        />
+
+                        <TableHeadersAxis<
+                          PickByType<TableHeadersConfig, Filter[][]>
+                        >
+                          id="rowGroups"
+                          legend="Move row headers"
+                          name="rowGroups"
+                          onMoveGroupToOtherAxis={groupIndex =>
+                            handleMoveGroupToOtherAxis(
+                              'rowGroups',
+                              groupIndex,
+                              form,
+                            )
+                          }
+                        />
+                      </FormGroup>
+                    </DragDropContext>
+
+                    <Button
+                      ariaControls={id}
+                      ariaExpanded
+                      className="govuk-!-margin-left-5 govuk-!-margin-top-3"
+                      type="submit"
                     >
-                      name="columnGroups"
-                      legend="Column groups"
-                      groupLegend="Column group"
-                    />
-                  </div>
-                </FormGroup>
-              </DragDropContext>
-
-              <Button type="submit">Re-order table</Button>
-            </Form>
-          );
-        }}
-      </Formik>
-    </Details>
+                      Update and view reordered table
+                    </Button>
+                  </Form>
+                );
+              }}
+            </Formik>
+          </div>
+
+          <div
```

Comment:
Just wondering if we want to not render a div if there is no `screenReaderMessage` set?

Reply:
We do, with `aria-live` screen readers looks for changes in an existing div (or whatever element the `aria-live` is on) to know whether to announce the changes.

Reply:
Ah right cool, that's good to know. Thanks